WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayTryPaymentPromptViewModel.swift

```diff
@@ -97,7 +104,8 @@ final class SetUpTapToPayTryPaymentPromptViewModel: PaymentSettingsFlowPresented
                                                                                   presentNoticeSubject: self.presentNoticeSubject,
                                                                                   analyticsFlow: .tapToPayTryAPayment),
                     siteID: self.siteID,
-                    orderID: order.orderID)
+                    orderID: order.orderID,
+                    order: order)
```

Comment:
Yeah. I left it in order to avoid having to go `summaryViewModel.order.orderID`, because a (simplistic) reading of the "Law" of Demeter is to avoid accessing properties more than one dot away, and that was in my head when I wrote this. 

Obviously we have plenty of examples of this kind of thing, and if we removed/renamed `orderID` on `Order`, we'd be in a world of hurt regardless of having a second dot here or not. 

I don't love the repetition in the access above, but it's not too bad, and the repetition in the initialisation of the ViewModel might somewhere an error could creep in during some future change or refactor, even though it's very simple. Perhaps a better fix would be to make the `orderID` property a computed passthrough, so we can still get it directly on the view model.

Reply:
Thanks for the explanations!

> "Law" of Demeter

TIL this has a name :D